### PR TITLE
ID Localization for access application cluster dahsboard web ui

### PR DESCRIPTION
**Pull request by mufti1**
adding bahasa translation for /docs/tasks/access-application-cluster/web-ui-dashboard/

issue #20111

changes:
- tasks, access-application-cluster folder
- web-ui-dashboard file

/language id

### CODE REVIEW

File: content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+Antarmuka _Dashboard_ secara standar tidak dirilis. Untuk merilisnya, kamu dapat menjalankan perintah berikut:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml
+```
+
+## Mengakses Antarmuka _Dashboard_
+
+
+Untuk melindungi data klaster kamu, perilisan _Dashboard_ menggunakan konfigurasi minimal _RBAC_ secara standar. Saat ini, _Dashboard_ hanya mendukung otentikasi dengan _Bearer Token_. Untuk membuat token untuk demo, kamu dapat mengikuti petunjuk kami [membuat sampel pengguna](https://github.com/kubernetes/dashboard/blob/master/docs/user/access-control/creating-sample-user.md).
```
[the remark this answers is not in the document]
Comment (mufti1) [reply content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md:35]: sesuatu yg relate ke kubernetes disini nyebutnya "kita" ya okay

File: content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+{{< warning >}}
+Sampel pengguna yang telah dibuat di tutorial akan memiliki hak istimewa sebagai _administrative_ dan hanya untuk tujuan pembelajaran.
```
[the remark this answers is not in the document]
Comment (mufti1) [reply content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md:38]: knp ada tambahan lah di hanya?

File: content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+## Tampilan selamat datang
+
+Ketika kamu mengakses _Dashboard_ di klaster yang kosong, kamu akan melihat halaman selamat datang. Halaman ini berisi tautan ke dokumen ini serta tombol untuk merilis aplikasi pertama kamu. Selain itu, kamu dapat melihat aplikasi sistem mana yang berjalan secara standar di [ruang nama](/docs/tasks/administer-cluster/namespaces/) `kube-system` dari klaster kamu, misalnya _Dashboard_ itu sendiri.
+
+![Kubernetes Dashboard welcome page](/images/docs/ui-dashboard-zerostate.png)
+
+## Merilis aplikasi yang sudah dikemas
+
+_Dashboard_ memungkinkan kamu untuk membuat dan merilis aplikasi yang sudah dikemas sebagai _Deployment_ dan _Service_ opsional dengan _simple wizard_. Kamu secara manual dapat menentukan detail aplikasi, atau mengunggah berkas YAML atau JSON yang berisi konfigurasi aplikasi.
```
[the remark this answers is not in the document]
Comment (mufti1) [reply content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md:64]: `Kamu manual dapat dengan manual` agak membingungkan kalau dibaca

**Comment by mufti1** [comment]
@phanama I've resolved your discussion

File: content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+  Service lainnya yang hanya dapat diakses dari dalam klaster disebut Service internal.
+
+  Terlepas dari jenis Service, jika kamu memilih untuk membuat sebuah Service dan Container kamu berjalan di  sebuah porta(arah masuk), kamu perlu menentukan dua porta. Service akan memetakan porta(arah masuk) ke porta target yang ada di sisi Container. Service akan mengarahkan ke Pod-Pod kamu yang sudah di-_deploy_. Protokol yang didukung adalah TCP dan UDP. Nama DNS internal untuk Service ini akan sesuai dengan nama aplikasi yang telah kamu tentukan diatas.
+
+Jika membutuhkan, kamu dapat membuka bagian **Advanced options** di mana kamu dapat menyetel lebih banyak pengaturan:
+
+- **Description**: Tels yang kamu masukkan ke sini akan ditambahkan sebagai sebuah [anotasi](/docs/concepts/overview/working-with-objects/annotations/) ke Deployment dan akan ditampilkan di detail aplikasi.
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md:90]: typo
```suggestion
- **Description**: Teks yang kamu masukkan ke sini akan ditambahkan sebagai sebuah [anotasi](/docs/concepts/overview/working-with-objects/annotations/) ke Deployment dan akan ditampilkan di detail aplikasi.
```

File: content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+- **_Run as priveleged_**: Pengaturan ini untuk menentukan sebuah proses yang berjalan dalam [_privileged container_](/docs/user-guide/pods/#privileged-mode-for-pod-containers) sepadan dengan proses yang berjalan sebagai _root_ pada _host_-nya. _Priveleged container_ dapat menggunakan kemampuan seperti memanipulasi _stack_ jaringan dan mengakses perangkat-perangkat. 
+
+- **_Environment variables_**: Kubernetes mengekspos Service melalui [_environment variable_](/docs/tasks/inject-data-application/environment-variable-expose-pod-information/). Kamu dapat membuat _environment variable_ atau meneruskan argumen ke perintah-perintah untuk menjalankan Container dengan nilai dari _environment variable_. _Environment Variable_ dapat digunakan pada aplikasi-aplikasi untuk menemukan sebuah Service. Nilai _environment variable_ dapat merujuk ke variabel lain menggunakan sintaksis `$(VAR_NAME)`.
+
+### Menggungah berkas YAML atau JSON
+
+Kubernetes mendukung pengaturan deklaratif. Dengan cara ini, semua pengaturan disimpan dalam bentuk berkas YAML atau JSON menggunakan skema sumber daya [[API](/docs/concepts/overview/kubernetes-api/).
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md:126]: typo, double `[`
```suggestion
Kubernetes mendukung pengaturan deklaratif. Dengan cara ini, semua pengaturan disimpan dalam bentuk berkas YAML atau JSON menggunakan skema sumber daya [API](/docs/concepts/overview/kubernetes-api/).
```

File: content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+Sebagai alternatif untuk menentukan detail aplikasi di _deploy wizard_, kamu dapat menentukan sendiri detail aplikasi kamu dalam berkas YAML atau JSON, dan mengunggah berkas tersebut menggunakan Dashboard.
+
+## Menggunakan Dashboard
+Bagian ini akan menjelaskan bagian-bagian yang ada pada Antarmuka Dashboard Kubernetes; apa saja yang mereka sediakan dan bagaimana cara menggunakanya.
+
+### Navigation
+
+Ketika ada objek Kubernetes yang sudah didefinisikan di dalam klaster, Dashboard akan menampilkanya di tampilan awalnya. Secara bawaan hanya objek-objek dalam Namespace _default_ saja yang ditampilkan di sini dan kamu dapat menggantinya dengan selektor Namespace yang berada di menu navigasi.
+
+Dashboard menampilkan jenis objek Kubernetes dan mengelompokanya dalam beberapa kategori menu.
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/web-ui-dashboard.md:137]: typo
```suggestion
Dashboard menampilkan jenis objek Kubernetes dan mengelompokkannya dalam beberapa kategori menu.
```

**Comment by girikuncoro** [comment]
Thanks @mufti1 for your contribution, great translation I hope you had the chance to play with the latest Dashboard 🎉 I have found couple of typos, you can address them in follow up PR. Let's get this in for now

/lgtm
/approve

**Pull request merged**